juniper_official/interface/check-ethernet-crc-framing-errors.rule

```diff
@@ -0,0 +1,217 @@
+/*
+ * Monitors Cyclic Redundancy,In-Fragment and Jabber frames errors and notifies when anomalies are found
+ * Two inputs control detection
+ *
+ *   1) interface-name-variable, matches the interfaces
```

Comment:
same as above, have same variable name("interface-name") in all rules

Reply:
Changed to interface-name